integreat_cms/api/urls.py

```diff
@@ -136,6 +136,11 @@
                     "<slug:language_slug>/wp-json/extensions/v3/",
                     include(content_api_urlpatterns),
                 ),
+                path(
+                    "<slug:language_slug>/wp-json/ig-mpdf/v1/pdf",
```

Comment:
```suggestion
                    "<slug:language_slug>/wp-json/ig-mpdf/v1/pdf/",
```
I think we should use the canonical url with trailing slash here - Django will then automatically redirect to this path if the trailing slash is missing (this doesn't work the other way round, so if the path is defined without trailing shlash, requesting the endpoint with trailing slash will lead to a 404. But ok, since it's a legacy endpoint, we probably don't need to support the url with trailing slash).